torch/_inductor/pattern_matcher.py

```diff
@@ -783,12 +783,15 @@ def is_start_of_fx_graph(graph, node):
     return node is next(iter(graph.nodes))
 
 
+_mutation_op_re = re.compile(r"_$|(\b|_)(set|enter|exit|seed)(\b|_)")
```

Comment:
Might be worth it to add a comment specifying which additional names we're looking for, i.e. `_set_grad_enabled`, `enter_functional_autocast`, `exit_functional_autocast`, `manual_seed`.